src/SelfService/Application/MembershipApplicationService.cs

```diff
@@ -281,7 +281,11 @@ public async Task LeaveCapability(CapabilityId capabilityId, UserId userId)
     [TransactionalBoundary, Outboxed]
     public async Task JoinCapability(CapabilityId capabilityId, UserId userId)
     {
-        _logger.LogInformation("User {userId} was directly added as a member of capability {capabilityId}", userId);
+        _logger.LogInformation(
+            "User {userId} was directly added as a member of capability {capabilityId}",
+            userId,
+            capabilityId
+        );
```

Comment:
When we use this logger we use capital letters for the fields:
`"User {UserId} was directly added as a member of capability {CapabilityId}"`

Its for filtering in cloudwatch i assume